library/core/src/num/int_macros.rs

```diff
@@ -2000,5 +2000,176 @@ macro_rules! int_impl {
         pub const fn max_value() -> Self {
             Self::MAX
         }
+
+        /// Adds to the current value, returning the previous value.
+        ///
+        /// This operation wraps around on overflow.
+        ///
+        /// # Examples
+        ///
+        /// ```
+        /// #![feature(int_fetch_ops)]
+        #[doc = concat!("let foo: ", stringify!($SelfT), " = 0;")]
+        /// assert_eq!(foo.fetch_add(10), 0);
+        /// assert_eq!(foo, 10);
+        /// ```
+        #[inline(always)]
+        #[unstable(feature = "int_fetch_ops", issue = "76904")]
```

Comment:
Rather than putting the wrong issue, can you put "none"? That makes it easier to catch later that it needs to be updated.
```suggestion
        #[unstable(feature = "int_fetch_ops", issue = "none")]
```

Reply:
Sure, I didn't know what this was a valid value to use here.